File: opcua-webservice-spring/core/src/main/java/org/opcfoundation/webapi/service/types/BrowseNextContext.java
package org.opcfoundation.webapi.service.types;

import org.eclipse.milo.opcua.stack.core.types.builtin.ByteString;
import org.eclipse.milo.opcua.stack.core.types.structured.RequestHeader;
import org.springframework.lang.Nullable;
import org.springframework.web.context.request.NativeWebRequest;

import java.util.List;

public class BrowseNextContext extends ServiceContext {
    private final List<ByteString> continuationPoints;
    private final boolean releaseContinuationPoints;

    public BrowseNextContext(
            List<ByteString> continuationPoints,
            boolean releaseContinuationPoints,
            @Nullable String serverUri,
            @Nullable RequestHeader header,
            @Nullable NativeWebRequest webRequest)
    {
        super(serverUri,header,webRequest);
        this.continuationPoints = continuationPoints;
        this.releaseContinuationPoints = releaseContinuationPoints;
    }

    public List<ByteString> getContinuationPoints() {
        return continuationPoints;
    }

    public boolean getReleaseContinuationPoints() {
        return releaseContinuationPoints;
    }
}

File: opcua-webservice-node/source/src/server/service/contexts/BrowseNextContext.ts
import { RequestHeader } from "opcua-webapi";
import { ServiceContext } from "./ServiceContext";

export class BrowseNextContext extends ServiceContext
{
    private _continuationPoints: Array<string>;
    private _releaseContinuationPoints: boolean;  
    
    constructor(
        releaseContinuationPoints: boolean,
        continuationPoints?: Array<string>,        
        serverUri?: string,
        requestHeader? : RequestHeader)
    {
        super(serverUri, requestHeader);
        this._releaseContinuationPoints = releaseContinuationPoints;
        this._continuationPoints = (continuationPoints) ? continuationPoints : [];        
    }

    get continuationPoints() : Array<string>
    {
        return this._continuationPoints;
    }

    get releaseContinuationPoints() : boolean
    {
        return this._releaseContinuationPoints;
    }
}